app.js

```diff
@@ -85,15 +113,19 @@ app.use("/sessions", require("./routes/sessionRoutes"));
 app.use("/tasks", auth, tasks);
 app.use("/events", auth, events);
```

Comment:
You enable the auth middleware for both `/events` and `/tasks`, which means that any request to those resources or any resources in those paths will need to include the Authorization header and token. Probably the best (and easiest to implement) solution is to only enable the auth middle for certain paths and operations in the routers.